adafruit_mcp2515/__init__.py

```diff
@@ -25,6 +25,11 @@
 """
 
 from collections import namedtuple
+
+try:
+    from typing_extensions import Literal
+except ImportError:
+    pass
```

Comment:
I would add this after `from .timer import Timer`, so that we can quickly see that these are typing imports only, since it comes after all the functional imports.

Reply:
Done on last commit.